packages/cli/src/register-command.ts

```diff
@@ -110,12 +111,18 @@ async function promptForMissing(cmd: Command, spec: CommandSpec, params: object)
         const details = await param.details?.(params);
         const values = [];
         for (const d of details) {
+          if (!params.interactive || DISABLE_INTERACTIVITY) {
+            throw new Error(`Missing required parameters ${name}`);
```

Comment:
I'd prefer if we collect all missing parameters and throw a single validation error - it's less frustrating to the user, as they know all at once every parameter they need to add

Reply:
I like this idea but I'm going to implement it in a different PR.

Reply:
https://github.com/OpenZeppelin/openzeppelin-sdk/issues/1398